docs/source/models/risk.rst

```diff
@@ -4,17 +4,100 @@
 Modeling Risk Exposure
 ======================
 
-.. todo::
+.. note::
 
-   Overview of modeling GBD risk factor exposure.
+   Vivarium has the infrastructure to model any risks in the GBD hierarchy. 
+   You should use tools from ``vivarium_inputs`` to pull and examine the 
+   data for your risks before including them in your model.
 
-.. contents:
+GBD Overview
+------------
 
 What is a risk factor?
-----------------------
+++++++++++++++++++++++
+A risk factor is any attribute, characteristic or exposure of an individual 
+that increases the likelihood of developing a disease or injury. Some examples 
+of the more important risk factors are underweight, unsafe sex, high blood pressure, tobacco and alcohol consumption, sanitation and hygiene, and unsafe water. [WHO]_ In GBD, a risk factor refers to any exposure that leads to a 
+loss of health in the population; it can be anywhere on the causal chain as 
```

Comment:
I would change "In GBD, a risk factor refers to **any** exposure that leads to a loss of health" to "In GBD, a risk factor refers to **a** exposure that leads to a loss of health" because the GBD list of risk factors is not an exhaustive list of risks that lead to health loss (this may be worth noting!)

Reply:
Thanks for pointing it out! The GBD 2019 risk factor list lives at: https://hub.ihme.washington.edu/display/GBD2019/GBD+2019+Risk+factors?preview=/54736328/80467412/list_of_risks_gbd_2019.xlsx